keywording/text.xml

```diff
@@ -299,10 +299,9 @@ Vulnerability Treatment Policy</uri>
 <body>
 
 <p>
-AMD64, X86: If you are the maintainer of a package that currently has open bugs
-for amd64 or x86 stabilization and own the respectively amd64 or x86 hardware,
-you can do your own testing and keywording of your packages; as long as it is
-not a core system set dependency.
+AMD64, X86: If you are the maintainer of a package and own the respective amd64
+or x86 hardware, you can do your own testing (stabilization and keywording) of
```

Comment:
The impression I get from x86 project page is that hardware is not a strict requirement, as long as you have a "pure" x86 "environment" (be it chroot, container, virtual machine...)
https://wiki.gentoo.org/wiki/Project:X86/Arch_Testers_FAQ#What_are_the_basic_system_requirements_if_any.3F
https://wiki.gentoo.org/wiki/Project:X86/Arch_Testers_FAQ#I_don.27t_run_stable_x86.2C_my_box_is_.7Ex86._How_can_I_setup_an_x86_chroot.3F
https://wiki.gentoo.org/wiki/Project:X86/Chroot_Guide

Reply:
Good point! What do you think of bfb4b63bbd19f0e5e71b3b11e89064ab08fd6298?

Reply:
Well as I tried to hint earlier, I'd call it "environment" because virtual machines etc are allowed... :)
(I've no clue about correct syntax for devmanual, but noticed that long urls get wrapped between
```
<uri
link="..."
```
which allows easier wrapping at 80 characters)